#!/usr/bin/env python
PACKAGE = 'cob_leg_detection'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('connection_threshold',     double_t,   0, '[m]', 0.06, 0, .25)
gen.add('min_points_per_group',     int_t,      0, '', 5, 1, 20)

exit(gen.generate(PACKAGE, 'trainingset_creator', 'TrainingSetCreator'))
